### PR TITLE
[WIP] added a community page

### PR DESCRIPTION
**Pull request by resskies**
added community page
fixed issue #46
WIP issue #23

### CODE REVIEW

**Comment by vck3000** [comment]
Please change the title of this Pull Request to include "[WIP]". When you're ready to remove it, please take it out. That way I know its ready to merge :). Thanks.

File: views/forum/show.ejs [hunk 1]
```diff
@@ -213,8 +213,8 @@
                         <% } %>
                         <p class="commentOptions">
                         
-                            <a class="reply">Reply</a>
-                            <a class="like " idOfElement=<%=comment._id%> idofcomment=<%=comment._id%> idOfForum=<%=forumThread._id%> typeofelement='comment'>
+                            <a class="reply" style="cursor: pointer">Reply</a>
+                            <a class="like" style="cursor: pointer" dOfElement=<%=comment._id%> idofcomment=<%=comment._id%> idOfForum=<%=forumThread._id%> typeofelement='comment'>
```
Comment (vck3000) [inline views/forum/show.ejs]: I think you accidentally deleted the 'i' in 'idOfElement' here.

Comment (resskies) [reply views/forum/show.ejs]: Oops, added it back

**Comment by resskies** [comment]
Closing pull request as I have made different branches for each fix.

**Pull request closed without merging**